transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -78,6 +79,9 @@ public Integer call() throws Exception {
     private volatile int wakenUp;
     private volatile int ioRatio = 50;
 
+    // See http://man7.org/linux/man-pages/man2/timerfd_create.2.html.
```

Comment:
Is this limit defined in the docs, or is this just based upon the size of the data types? If its data type based this may not be correct and depends upon the type widths. For example `long` may be unsigned 32, but IIRC it is only guaranteed to be smaller than `int`. `time_t` maybe 32 or 64 byte signed integer. If this is based upon datatypes we should get this information via JNI ... hopefully in a limits file somewhere?

make this a java doc comment?

Reply:
its defined in the manpage to be the max you can set.

Reply:
> EINVAL new_value is not properly initialized (one of the tv_nsec
              falls outside the range zero to 999,999,999).

Gotcha ... consider adding to javadocs.